GeneralsMD/Code/GameEngine/Include/Common/Money.h

```diff
@@ -64,13 +64,24 @@ class Money : public Snapshot
 
 public:
 
-	inline Money() : m_money(0), m_playerIndex(0)
+	inline Money() : m_money(0), m_playerIndex(0), m_startingCash(0), m_currentBucket(0), m_lastBucketFrame(0)
 	{
+		for (UnsignedInt i = 0; i < 60; ++i)
+		{
+			m_incomeBuckets[i] = 0;
+		}
 	}
 
 	void init()
 	{
 		m_money = 0;
```

Comment:
It looks like `init()` is now synonymous to `setStartingCash(0)`, so that can be simplified.